react/index.js

```diff
@@ -223,6 +223,12 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
+    const getBuyButtonVisibility = () => {
+      return !((showButtonOnHover || showBuyButton) && showBuyButton) ? 'hidden' : 'visible'
+    }
+
+    const style = { visibility: getBuyButtonVisibility() }
```

Comment:
Use `tachyons`'s classes instead. `classname` could do it!

Reply:
`tachyons` don't support change the `visibility` attribute.